### ISSUE TITLE
Signal/Transit peptides in Reactome (12 confirmed cases)

### ISSUE THREAD

**Issue by nataled** [geneontology/pathways2GO#304]
The following is the list of signal or transit peptides given as EWASes in Reactome:

1-58 of O00116 is a transit peptide (Reactome:R-HSA-9033519)
1-26 of P09110 is a transit peptide (Reactome:R-HSA-9033534)
1-47 of P06576 is a transit peptide (Reactome:R-HSA-8986172)
1-52 of Q9H4I9 is a transit peptide (Reactome:R-HSA-8949642)
1-28 of O95169 is a transit peptide (Reactome:R-HSA-8986166)
1-34 of Q96H96 is a transit peptide (Reactome:R-HSA-8986157)
1-32 of P00480 is a transit peptide (Reactome:R-HSA-8986137)
1-30 of O14832 is a transit peptide (Reactome:R-HSA-9033518)
1-26 of P10809 is a transit peptide (Reactome:R-HSA-8986171)
1-41 of Q16595 is a transit peptide (Reactome:R-HSA-8986117)
1-61 of P05496 is a transit peptide (Reactome:R-HSA-8986143)
1-25 of P01160 is a signal peptide (Reactome:R-HSA-5578791)

Note that these were found by cross-checking the signal/transit peptide annotation in UniProtKB against the sequence range given in Reactome. Thus, only perfect matches will be found. In other words, if there is a difference (such as if Reactome has an old transit peptide sequence range 1-30 but UniProtKB changed it to 1-31) then I won't be able to find it.

At least two of these (R-HSA-5578791 and R-HSA-8986143; last two on the list) seem to be just an output of proteolytic processing.

**Comment by deustp01** [comment]
Indeed, Reactome has not annotated any role for any of the aminoterminal peptide fragments on this sample list, and for most of the ones associated with proteins that are synthesized in the cytosol and translocated to the mitochondrial matrix, Reactome has annotated the degradation of the aminoterminal peptide once inside the mitochondrion.

I'm passing the list on to Reactome curators to discuss how we want to annotate such targeting peptides (and thus  whether they need separate proteoform IDs).

**Comment by MarijaOM** [comment]
In my opinion, having two subclasses of cleavage reactions - one where all cleavage products are shown (because all participate in downstream events) and another where only some are shown, would be confusing to curators and users and inconsistent from the standpoint of our data model. Also, the class of black box events would be overloaded by reactions whose products are clearly experimentally defined and simple to annotate, but we choose not to do it. In addition, we do provide coordinates and controlled vocabulary names for all cleavage fragments, so they can be clearly distinguished from other products of posttranslational processing that refer to the same UniProt entry.

**Comment by deustp01** [comment]
To summarize the discussion, we mostly do not annotate the cleavage of a proprotein to generate a mature protein plus a leader polypeptide. But we make exceptions when we want to annotate the protease doing the cleaving or when the cleavage event is part of a reaction sequence that couples cleavage to other events such as translocation of the maturing protein. The total number of exceptions is small, so my opinion is that, so long as we get the coordinates of the proprotein and its cleavage products correctly and consistent with UniProt numbering, there's no harm in creating PRO IDs for this small number of leader peptides.

Now CLOSING, in the hope that this resolution is OK
